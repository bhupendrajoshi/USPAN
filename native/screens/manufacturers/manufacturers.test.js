import 'react-native';
import React from 'react';
import Manufacturers from './manufacturers';

import renderer from 'react-test-renderer';

describe('Manufacturers component', () => {
    test('it should render correctly', () => {
        const tree = renderer.create(
            <Manufacturers />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
