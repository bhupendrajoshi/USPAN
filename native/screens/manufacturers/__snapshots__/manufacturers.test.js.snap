// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manufacturers component it should render correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Manufacturers
  </Text>
</View>
`;
